thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomalydetection/model/merge/MinMaxThresholdMergeModel.java

```diff
@@ -0,0 +1,67 @@
+package com.linkedin.thirdeye.anomalydetection.model.merge;
```

Comment:
I am wondering if the mergemodel is properly defined. 1) can this be called independent of 'AnomalyDetectionFunction'. this use AnomalyDetectionContext to update a merged anomaly, which seems odd. Do we actually store this context in our anomaly results? 2) I did not see where you provide this MergedAnomalyResult.